### PR TITLE
chore: Auto generate colors via SwiftGen 🎨

### PR DESCRIPTION
**Pull request by abdelhamid-f-nasser**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature

## Description
- Adds generated 🎨 UI-Colors using  🧬swiftGen .
- Note: Moved the colors to a separate assets bundle.
- Add all default colors 🎨
- Add system colors ( to expose the colors to our components for better readability)

## Related Tickets & Documents
Cannot access Jira to find ticket number

## Added tests?
- [x] 🙋 no, because I need help

## Added to documentation?
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
Robot fails

### CODE REVIEW

File: Healthy/Classes/Generated/UIColors+Generated.swift [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        UIColor(named: "Gray 2")!
+      }
+
+      static var gray3: UIColor {
+        UIColor(named: "Gray 3")!
+      }
+
+      static var gray4: UIColor {
+        UIColor(named: "Gray 4")!
+      }
```
Comment (ahmdmhasn) [inline Healthy/Classes/Generated/UIColors+Generated.swift]:
```suggestion
      static var black: UIColor {
        UIColor(named: "Black")!
      }

      static var gray1: UIColor {
        UIColor(named: "Gray 1")!
      }

      static var gray2: UIColor {
        UIColor(named: "Gray 2")!
      }

      static var gray3: UIColor {
        UIColor(named: "Gray 3")!
      }

      static var gray4: UIColor {
        UIColor(named: "Gray 4")!
      }
```

Let's update black to have the same color incremental values. ex. 20-40-60-80-100(Pure Black)

**Review by ahmdmhasn — APPROVED** [review]
🚀

File: Healthy/Classes/Generated/UIColors+Generated.swift (hunk 1 above)
Comment (abdelhamid-f-nasser) [reply Healthy/Classes/Generated/UIColors+Generated.swift]: Just to be sure, Should we add shades to the black from black 20 to black 100? and should I also do this to the rest of the colors?

**Review by Ahattalla — APPROVED** [review]
Good job ya bro ❤️

File: Healthy/Classes/Generated/UIColors+Generated.swift (hunk 1 above)
Comment (abdelhamid-f-nasser) [reply Healthy/Classes/Generated/UIColors+Generated.swift]: Oh so we change the grays to black 20-80 and the current black to Black 100?

Comment (ahmdmhasn) [reply Healthy/Classes/Generated/UIColors+Generated.swift]: Yes exactly, just the black.

**Comment by ahmdmhasn** [comment]
@abdelhamid-f-nasser Please make sure to drop all the irrelevant changes.

**Review by ahmdmhasn — APPROVED** [review]

**Pull request merged**